components/frontend/src/source/Sources.js

```diff
@@ -51,6 +52,14 @@ export function Sources({ reports, report, metric, metric_uuid, measurement, cha
     const sourceUuids = Object.keys(metric.sources).filter((sourceUuid) =>
         dataModel.metrics[metric.type].sources.includes(metric.sources[sourceUuid].type)
     );
+
+    const reload_source = (json) => {
+        if (json.affected !== undefined) {
+            show_message("success", `${json.affected} source(s) affected.`)
```

Comment:
Affected is very abstract. How about updated?

```suggestion
            show_message("success", `${json.nr_source_updated} source(s) updated.`)
```